src/utils/backend/redis.ts

```diff
@@ -1,9 +1,11 @@
 import IORedis from 'ioredis';
 
-export const redis = new IORedis({
-    host: 'redis',
-    password: 'RADAR',
-    port: 6379,
-    family: 4,
-    db: 0,
-});
+export function getRedis() {
```

Comment:
Can we restore constant please? In this case we'll initialize ioredis multiple times

Reply:
I had problems with only one single instance. Especially because i need one that listen and one that recieve in the same file. That did not work with the previous const. But i can try again.

Reply:
Yes please, because that is weird. If it's impossible in ioredis, you can leave it as is